### DISCUSSION TITLE
Can I import a .mojo file in the notebook?

### DISCUSSION THREAD

**Discussion by czheo** [modular/modular#166] [Q&A]
Can I put my code in a `.mojo` file and import it from the notebook?
I tried to create `MyTest.mojo` in the same dir like below.
```py
fn my_test():
    print(42)
```
And import in the notebook looks like
```
from MyTest import my_test

error: Expression [1]:5:20: unable to locate module 'MyTest'
from MyTest import my_test
                   ^
```

**Comment by mfranzon** [comment]
Seems that it is not possible in the notebook. Relative imports are not implemented:
```traceback
error: Expression [31]:6:6: TODO: relative package imports are not yet supported
from .test import my_test
```
And if my understanding is correct, all the available modules are into `/.modular/Kernels/mojo/` folder, (permissions 755, so no changes available ) .

**Comment by Mogball** [comment]
Please file a feature request!

**Reply by czheo** [reply]
feature request created https://github.com/modularml/mojo/issues/174
